To: vim_dev@googlegroups.com
Subject: Patch 7.3.303
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.303 (after 7.3.296)
Problem:    Compilation error.
Solution:   Correct return type from int to pid_t. (Danek Duvall)
Files:	    src/os_unix.c


*** ../vim-7.3.302/src/os_unix.c	2011-09-07 15:04:26.000000000 +0200
--- src/os_unix.c	2011-09-08 20:47:47.000000000 +0200
***************
*** 159,165 ****
  #else
  typedef int waitstatus;
  #endif
! static int  wait4pid __ARGS((pid_t, waitstatus *));
  
  static int  WaitForChar __ARGS((long));
  #if defined(__BEOS__)
--- 159,165 ----
  #else
  typedef int waitstatus;
  #endif
! static pid_t wait4pid __ARGS((pid_t, waitstatus *));
  
  static int  WaitForChar __ARGS((long));
  #if defined(__BEOS__)
*** ../vim-7.3.302/src/version.c	2011-09-08 23:22:35.000000000 +0200
--- src/version.c	2011-09-08 23:23:43.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     303,
  /**/

-- 
For humans, honesty is a matter of degree.  Engineers are always honest in
matters of technology and human relationships.  That's why it's a good idea
to keep engineers away from customers, romantic interests, and other people
who can't handle the truth.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
